Remove Debian-specific names, this breaks on other systems.
The python library name is passed by robotpkg anyway.

--- cmake/boost.cmake~	2018-05-12 11:21:52.000000000 +0200
+++ cmake/boost.cmake	2018-05-15 13:50:19.741275462 +0200
@@ -67,7 +67,7 @@
         ENDIF(NOT PYTHONLIBS_FOUND)
 
         LIST(REMOVE_AT BOOST_COMPONENTS_ ${PYTHON_IN_BOOST_COMPONENTS})
-        SET(BOOST_PYTHON_COMPONENT "python${PYTHON_VERSION_MAJOR}${PYTHON_VERSION_MINOR}")
+        SET(BOOST_PYTHON_COMPONENT "python")
         LIST(APPEND BOOST_COMPONENTS_ ${BOOST_PYTHON_COMPONENT})
     ELSE(Boost_VERSION VERSION_GREATER 106699)
       # Check if Python has been found
@@ -77,9 +77,9 @@
       IF(${PYTHON_VERSION_MAJOR} EQUAL 3) 
         LIST(REMOVE_AT BOOST_COMPONENTS_ ${PYTHON_IN_BOOST_COMPONENTS})
         IF(${UNIX})
-          SET(BOOST_PYTHON_COMPONENT "python-py${PYTHON_VERSION_MAJOR}${PYTHON_VERSION_MINOR}")
+          SET(BOOST_PYTHON_COMPONENT "python")
         ELSE(${UNIX})
-          SET(BOOST_PYTHON_COMPONENT "python${PYTHON_VERSION_MAJOR}${PYTHON_VERSION_MINOR}")
+          SET(BOOST_PYTHON_COMPONENT "python")
         ENDIF(${UNIX})
         LIST(APPEND BOOST_COMPONENTS_ ${BOOST_PYTHON_COMPONENT})
       ELSE(${PYTHON_VERSION_MAJOR} EQUAL 3) 
